tests/tests_default.yml

```diff
@@ -42,5 +43,8 @@
           register: result
           failed_when: result.stat.exists
 
+        - name: test - ensure cockpit_port is configured for firewall and selinux
+          include_tasks: tasks/check_firewall_selinux.yml
```

Comment:
Note that the default Fedora and RHEL configuration is fine by itself -- this should *not* mangle anything in the default case. The tasks file does not do anything harmful, so it's fine to call it here; I just wanted to point it out.